src/components/RichSelect/__stories__/index.stories.tsx

```diff
@@ -52,9 +52,13 @@ Uncontrolled.decorators = [
 export const Controlled = Template.bind({})
 Controlled.decorators = [
   () => (
-    <ControlValue value="">
+    <ControlValue<string> value="">
       {({ value, onChange }) => (
-        <RichSelect name="controlled" value={value} onChange={onChange}>
+        <RichSelect
+          name="controlled"
+          value={value}
+          onChange={newValue => newValue && onChange(newValue)}
```

Comment:
```suggestion
          onChange={newValue => newValue && onChange(newValue) as unknown as string}
```

Reply:
Meh, force casting should not be the solution right ?

Reply:
This is what you've done here: https://github.com/scaleway/scaleway-ui/pull/1378/files#diff-401752b10b3fb35cfa63400a966be9aef2b40e276df59abfa2007d1065136623R132

Btw I made a mistake, I wanted to type cast `newValue`, not `onChange`